src/main/java/org/hydev/mcpm/client/interaction/ProgressBar.java

```diff
@@ -113,7 +113,7 @@ private void forceUpdate()
         if (istty) cu.curUp(activeBars.size());
         activeBars.forEach(bar -> {
             cu.eraseLine();
-            out.println(bar.toString(theme, cols));
+            out.print(bar.toString(theme, cols));
```

Comment:
Why did this get changed

Reply:
I was trying to see why downloading a single file was creating a new progress bar each time the download progressed, but if that bug has already been sorted out, you can just ignore this change